consensus/src/aggregation/mod.rs

```diff
@@ -0,0 +1,1222 @@
+//! Recover threshold signatures to represent agreement over an ordered sequence of items.
```

Comment:
nit: `Recover threshold signatures over an externally synchronized sequence of items.`